internal/command/test_test.go

```diff
@@ -1426,3 +1427,134 @@ operation, and the specified output value is only known after apply.
 	}
 
 }
+
+// This test takes around 10 seconds to complete, as we're testing the progress
+// updates that are printed every 2 seconds. Sorry!
```

Comment:
I think this is fine given our timelines. If this becomes annoying in future we might be able to do something similar to the UI/JSON hook tests: 

https://github.com/hashicorp/terraform/blob/8360d7566ed93601ab9c6e347c8854cead9e330f/internal/command/views/hook_json_test.go#L27-L31
https://github.com/hashicorp/terraform/blob/8360d7566ed93601ab9c6e347c8854cead9e330f/internal/command/views/hook_json_test.go#L63-L69